sandbox/StatusInputPage.qml

```diff
@@ -66,6 +67,22 @@ Column {
         input.placeholderText: "Placeholder"
     }
 
+    StatusInput {
+        label: "Label"
+        charLimit: 30
+        input.placeholderText: "Input with validator"
+
+        validators: [
+            StatusMaxLengthValidator { maxLength: 30 }
```

Comment:
i think it should be done by `maximumLength` property. Looks like user can add text more than 30 chars but its not acceptable. When this option can be helpful?

Reply:
Yes, as explained above, this is to have room for a validator to kick in. Will double check with designs, but happy to roll back on the max length validator (which isn't needed anymore  if we have the limit enforced built-in)

Reply:
This is addressed here: https://github.com/status-im/StatusQ/pull/321